types/pi-wifi/index.d.ts

```diff
@@ -0,0 +1,186 @@
+// Type definitions for pi-wifi 1.2
+// Project: https://github.com/matrix-io/pi-wifi
+// Definitions by: Kent Loofbourrow <https://github.com/loofkid>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace piWifi;
+
```

Comment:
this is when package is kind of UMD module:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html#optional-global-usage
while this package looks like good, old CJS:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html#common-commonjs-patterns
(if you change this, you would have to add proper import in tests)

Reply:
I believe I've fixed this. Thank you for helping me get this right, it's my first (official) type definition.